codegen/cmd.go

```diff
@@ -184,9 +190,35 @@ func (c Command) compileProtos(grp *render.Group) error {
 	// set the descriptors on the group for compilation
 	grp.Descriptors = descriptors
 
+	for i, resource := range grp.Resources {
+		// attach the proto messages for spec and status to each resource
+		// these are processed by renderers at later stages
+		addMessagesToResource(descriptors, &resource)
+		grp.Resources[i] = resource
+	}
+
 	return nil
 }
 
+func addMessagesToResource(descriptors []*skmodel.DescriptorWithPath, resource *model.Resource) {
+	for _, fileDescriptor := range descriptors {
+
+		if fileDescriptor.GetPackage() == resource.Group.Group {
```

Comment:
Can't there potentially be many files in with the group's package?

Reply:
fixed (this would overwrite with `nil` if there is more than 1 file and last file does not contain message)